webapi/payfee.go

```diff
@@ -216,7 +216,13 @@ findAddress:
 		feeTxHash, err := dcrdClient.SendRawTransaction(payFeeRequest.FeeTx)
 		if err != nil {
 			log.Errorf("SendRawTransaction failed: %v", err)
+			log.Infof("Removing fee transaction for ticket %v", ticket.Hash)
 			sendErrorResponse("dcrwallet RPC error", http.StatusInternalServerError, c)
+			ticket.FeeTxHex = ""
```

Comment:
I think this needs to be reshuffled a little bit. We shouldn't set the feetx in the database and then un-set it. If the ticket is already confirmed, and broadcasting the fee tx fails, then we actually don't need to update the database at all - we dont need to store the wif, the hex or the vote choices. Just return an error and let the client try again with a new tx.